main.tf

```diff
@@ -76,6 +77,7 @@ resource "azurerm_kubernetes_cluster" "this" {
   ## Resources that only support UserAssigned
   dynamic "identity" {
     for_each = local.managed_identities.user_assigned
+
```

Comment:
I think this is here because we did not merge PR #105 ?

Reply:
hehehe strange